CHANGELOG.md

```diff
@@ -13,6 +13,14 @@ New features:
 
 * MRI sources have been updated to version 2.3.6.
 
+* `equal?` on foreign objects will check if the underlying objects are equal
+  if both are Java interop objects.
```

Comment:
"are equal" is a bit imprecise. "are the same instance" would be more precise.
But maybe it doesn't matter so much in the ChangeLog, the documentation below is clear.